From 9ddbefb486be45762e8fcab731b5de1c710507e0 Mon Sep 17 00:00:00 2001
From: Dimitrios Apostolou <jimis@gmx.net>
Date: Wed, 19 Dec 2012 22:32:52 +0100
Subject: [PATCH 162/204] socket: Too many incoming connections

Hello list,

some socket activated service gave me the error message you can see on
the subject, maybe systemd should be more verbose in that case.

Thanks,
Dimitris
(cherry picked from commit 80cba3795da3a43b27cbb219d7daf7d6f049b079)
---
 src/core/socket.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/socket.c b/src/core/socket.c
index 54377d6..5acf653 100644
--- a/src/core/socket.c
+++ b/src/core/socket.c
@@ -1441,7 +1441,7 @@ static void socket_enter_running(Socket *s, int cfd) {
                 Service *service;
 
                 if (s->n_connections >= s->max_connections) {
-                        log_warning("Too many incoming connections (%u)", s->n_connections);
+                        log_warning("%s: Too many incoming connections (%u)", UNIT(s)->id, s->n_connections);
                         close_nointr_nofail(cfd);
                         return;
                 }
-- 
1.8.1.5

